src/main/java/org/jboss/set/pull/processor/data/DefinedLabelItem.java

```diff
@@ -25,7 +25,8 @@ public String getLabel() {
         Missing_upstream_issue("Missing upstream issue"),
         Missing_issue("Missing issue"),
         Missing_upstream_PR("Missing upstream PR"),
-        Corrupted_upgrade_meta("Corrupted upgrade meta");
+        Corrupted_upgrade_meta("Corrupted upgrade"),
+        Missing_upgrade_meta("Missing upgrade");
```

Comment:
The new Missing_upgrade_meta is not used ?

Reply:
yep, missed it when I was cleaning